ipctest/src/main.rs

```diff
@@ -65,12 +67,78 @@ fn run() -> Result<()> {
     Ok(())
 }
 
+#[cfg(not(windows))]
+fn run_client(_pid: u32, _handle: usize) -> Result<()> {
+    bail!("not used on non-Windows");
+}
+
+#[cfg(windows)]
+fn run() -> Result<()> {
+    let handle = server::audioipc_server_start();
+    let fd = server::audioipc_server_new_client(handle);
+
+    let args: Vec<String> = std::env::args().collect();
+
+    let mut child = std::process::Command::new(&args[0])
+        .arg("--client")
+        .env("PID", format!("{}", std::process::id()))
+        .env("HANDLE", format!("{}", fd as usize))
+        .spawn()
+        .expect("child process failed");
+
+    child.wait().expect("child process wait failed");
+
+    server::audioipc_server_stop(handle);
+
+    Ok(())
+}
+
+#[cfg(windows)]
+fn run_client(pid: u32, handle: usize) -> Result<()> {
+    #[cfg(windows)]
```

Comment:
the `run_client` will only be compiled when the platform is Windows, so the inner `#[cfg(windows)]` could be removed.